src/main/RendererToMainProcessor.ts

```diff
@@ -385,27 +385,27 @@ async function displayOpenFileDialog (
   return selectedFile;
 }
 
-function resolveMessagePath (path: string, resolveIn?: string): string {
-  let fullPath = path;
+function resolveMessagePath (inputPath: string, resolveIn?: string): string {
+  if (path.isAbsolute(inputPath))
```

Comment:
Even though resolveXXXFilePath do this check we need to perform because of resolveIn segment that are joined with the input path below.